modules/heat_conduction/test/tests/meshed_gap_thermal_contact/tests

```diff
@@ -7,12 +9,17 @@
     # It's possible for this test to hit the "negative temperature" error
     # because the test uses values 0 and 1 for boundary conditions and
     # the solver may find slightly negative values in this case.
+    requirement = "The ThermalContact system shall enforce heat transfer
+    across a meshed gap in a 2D plane geometry."
   [../]
 
   [./annulus]
     type = 'Exodiff'
     input = 'meshed_annulus_thermal_contact.i'
     exodiff = 'meshed_annulus_thermal_contact_out.e'
+    rel_err = 5e-2
```

Comment:
This is pretty loose.  I'm guessing there is one variable that is pushing this that high. Does an exodiff file make sense for this case?